autogen/misc_utils.py

```diff
@@ -0,0 +1,16 @@
+import asyncio
+import sys
+
+
+async def ainput(prompt: str) -> str:
```

Comment:
The convention in this repo is to use `a_*` for names of `async` functions.

Reply:
fixed